### DISCUSSION TITLE
ReduceMap axes in Python

### DISCUSSION THREAD

**Discussion by SamWilsn** [microsoft/onnxruntime#27149] [API Q&A]
Hey! Complete ML beginner here, so forgive my bumbling around.

I'm attempting to export a training version of my pytorch model, where I use cosine similarity in my loss function.

When using `onnxruntime.training.onnxblock.blocks.ReduceSum`, how do you specify your axes? Or better yet, is there a way to avoid reimplementing the loss function?
